src/mocks/fixtures.ts

```diff
@@ -0,0 +1,152 @@
+import {SubmissionMetadata} from "../api";
+
+export const submissions: SubmissionMetadata[] = [
+  {
```

Comment:
@pkalita-lbl, will you add a comment about how you generated this example object / how someone might be able to generate an example object in the future?